sql/core/src/test/resources/sql-tests/inputs/misc-functions.sql

```diff
@@ -0,0 +1,10 @@
+-- test for misc functions
+
+-- typeof
+select typeof(null);
+select typeof(true);
+select typeof(1Y), typeof(1S), typeof(1), typeof(1L);
+select typeof(cast(1.0 as float)), typeof(1.0D), typeof(1.2);
+select typeof(date '1986-05-23'),  typeof(timestamp '1986-05-23'), typeof(interval '23 days');
+select typeof(x'ABCD'), typeof('SPARK'), typeof(cast('SPARK' as char(10))), typeof(cast('SPARK' as varchar(10)));
```

Comment:
`cast('SPARK' as char(10))` this is testing the type alias, it's not related to `typeof`, let's remove them.